### PR TITLE
Contributing: Fix links

### PR DESCRIPTION
**Pull request by Simran-B**
### Why:

Some of the links in CONTRIBUTING.md are broken.

### What's being changed:

Fix two typos and change anchor link to forking section, which does not have dedicated headlines for GitHub Desktop and command-line.

There are a couple more dead anchor links, but it looks like the content does not exist yet?

```md
- [Resolving merge conflicts](#resolving-merge-conflicts)
  - [In the GitHub user interface](#in-the-github-user-interface)
    - [Editing the file and committing the changes](#editing-the-file-and-committing-the-changes)
- [Troubleshooting](#troubleshooting)
  - [Failed status checks](#failed-status-checks)
- [Draft pull requests](#draft-pull-requests)
```

### Check off the following:

- [ ] ~All of the tests are passing.~
- [ ] ~I have reviewed my changes in staging.~
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -63,8 +63,7 @@ Now that you're a part of the GitHub Docs community, you can keep participating
 - [Starting with an issue](#starting-with-an-issue)
   - [Labels](#labels)
 - [Opening a pull request](#opening-a-pull-request)
-  - [Fork using GitHub Desktop](#fork-using-github-desktop)
-  - [Fork using the command line](#fork-using-the-command-line)
+  - [Fork using GitHub Desktop or the command line](#ready-to-make-a-change-fork-the-repo)
```
Comment (janiceilene) [inline CONTRIBUTING.md:66]:
```suggestion
```
We moved this information up to the top as well, so we can delete this line.

**Review by janiceilene — CHANGES_REQUESTED** [review]
Thanks for catching these @Simran-B! I added one suggestion. Once you accept that suggestion, we can review it again and get this merged in ⚡

**Comment by Simran-B** [comment]
You're welcome @janiceilene. Applied your suggestion. Can be merged, but you may want to address the remaining dead anchor links in another PR one way or the other.

**Comment by janiceilene** [comment]
@Simran-B Thanks for pointing those out! We did some last minute restructuring of this doc and must have missed the table of contents 😅 Would you open an issue about the broken links you found? Maybe someone else will get to fixing them before I do 🤞 

I'll merge this in as soon as the branch is updated and all of the tests are green 💚

**Review by janiceilene — APPROVED** [review]

**Comment by janiceilene** [comment]
If you haven't already, you can add yourself to [the list of contributors](https://github.com/github/docs#contributors-) by creating a new comment in this PR using [these instructions](https://allcontributors.org/docs/en/bot/usage#commands). Thanks again! :sparkles:

**Comment by Simran-B** [comment]
@all-contributors please add Simran-B for doc

**Comment by Simran-B** [comment]
Sorry, wrong user 😞 

@all-contributors please add @Simran-B for doc

**Pull request merged**